examples/kubernetes-basic-example/README.md

```diff
@@ -0,0 +1,40 @@
+# Kubernetes Basic Example
```

Comment:
Nice 👍

Reply:
BTW, you may want to add to our newsletter that Terratest now supports automated testing for Kubernetes as a top-level item. SEO gold.

Reply:
Yup was thinking of doing that as soon as this merges